src/components/puck/CtaWrapper.tsx

```diff
@@ -78,9 +78,9 @@ export const CTAWrapperComponent: ComponentConfig<CTAWrapperProps> = {
     entityField: {
       field: "",
       constantValue: {
-        label: "Call to Action",
+        name: "Call to Action",
```

Comment:
Not you but the this shouldn't be variant and fontSize, it should be label and link